### PR TITLE
Cramer's V to check correlation between nominal variables

### PR DESCRIPTION
**Pull request by elisehuard**
One of the possible measures (I believe there are a number of others). based on this https://en.wikipedia.org/wiki/Cram%C3%A9r's_V

### CODE REVIEW

File: test/kixi/stats/core_test.cljc [hunk 1]
```diff
@@ -172,6 +172,29 @@
           (when-not (zero? d)
             (/ (reduce + (map * mxs mys)) d)))))))
 
+(defn cramers-v'
+  [fx fy coll]
+  (let [count-values (fn [m]
+                       (into {} (for [[k v] m] [k (count v)]) ))
+        x-counts (count-values (group-by fx coll))
```
Comment (henrygarner) [inline test/kixi/stats/core_test.cljc]: My sense is that `{x|y|xy}-counts` could be calculated in a single `reduce` which could replace the 6 iterations with 1

File: test/kixi/stats/core_test.cljc [hunk 2]
```diff
@@ -441,26 +464,33 @@
   test-opts
   ;; Take maps like {}, {:x 1}, {:x 2 :y 3} and compute correlation matrix
   (for-all [coll (gen/vector (gen/map (gen/elements [:x :y :z]) gen/int))]
-    (is (=ish (transduce identity (kixi/covariance-matrix {:x :x :y :y :z :z}) coll)
-              (covariance-matrix' coll)))))
+           (is (=ish (transduce identity (kixi/covariance-matrix {:x :x :y :y :z :z}) coll)
```
Comment (henrygarner) [inline test/kixi/stats/core_test.cljc]: Whitespace. I think I must have updated my .emacs.d to set the indentation for `for-all`. I reckon it's better without the additional space (+ occurrences below)

File: src/kixi/stats/core.cljc [hunk 3]
```diff
@@ -296,6 +296,41 @@
...
+  [fx fy]
+  (fn
+    ([] [{} {} {} 0])
+    ([[f1 f2 f12 n] row]
+     (let [k1 (fx row)
+           k2 (fy row)
+           k12 [k1 k2]
+           increment-count (fn [m k]
+                             (if (get m k)
+                               (update m k inc)
```
Comment (elisehuard) [inline src/kixi/stats/core.cljc]: cool

File: test/kixi/stats/core_test.cljc (hunk 2 above)
Comment (elisehuard) [inline test/kixi/stats/core_test.cljc]: will fix, thank you

**Pull request merged**